LICENSE

```diff
@@ -1,4 +1,8 @@
-Copyright (c) 2013-2015, Alfred Klomp
+Copyright (c) 2005-2007, Nick Galbreath
+Copyright (c) 2013-2017, Alfred Klomp
+Copyright (c) 2015-2017, Wojciech Mula
+Copyright (c) 2016-2017, Matthieu Darbois
```

Comment:
Totally fine to put you and @WojciechMula here, but does "Nick Galbreath" know that he is being mentioned here? Did he agree for his code to be a part of this project?